email.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Email": "Placeholder_for_Y/N",
+  "Sender": "Placeholder_for_str",
+  "smtp_server": "Placeholder_for_str",
+  "smtp_port": "Placeholder_for_int"
+}
```

Comment:
I'm not sure what this file is doing, here. A little more info on its purpose, please?

Reply:
email.json is meant to hold specific email configuration data, such as who should be the sender, what the smtp server should be, etc. I have the "Placeholder_for" to try to hint at what should be put there, for a user to then change. I was going to include a more in-depth explanation in an updated readme. That being said, if I'm making the change mentioned above, then I can also change from using a configuration file to using command line arguments for email configuration. I had originally done this, but eventually shied away from it thinking it was the cause of errors I was facing, and decided to keep my changes anyway when I realized the email arguments were not the root cause of the errors. I think I will change the email to using command line arguments instead, as it feels to me to go hand in hand with the above change I think is for the best about flagging/state.